desktop/components/credit_card/test/error_handling_form.coffee

```diff
@@ -66,3 +66,13 @@ describe 'FavoritesStatusModalView', ->
     it 'adds stripe errors', ->
       @errorHandlingForm.showError 'description', { status: 400, error: { additional: 'additional info'} }
       $('.error').text().should.equal 'Your card appears to be missing or malformed. Please try another card or contact support. additional info'
+
+    it 'handles @errors properties that are a function (that return an errors object)', ->
+      @errorHandlingForm.errors = -> {"Meow meow meow": 'Sorry, this endpoint is for cats'}
+      @errorHandlingForm.showError 'description', { responseText: "{ \"type\": \"param_error\", \"message\": \"Meow meow meow\" } " }
+      $('.error').text().should.equal 'Sorry, this endpoint is for cats'
+
+    it 'handles error messages that are a function', ->
```

Comment:
Can you write a test that checks to see if it can still handle errors that are a string (for backwards compatibility)?